templates/site/index.html

```diff
@@ -2,27 +2,53 @@
 {% set page_title = "The DACCS Project" %}
 {% set page_heading = "The DACCS Project" %}
 
-{% block project_description %}
+{% block content %}
 <div class="col-12 d-block py-3 mx-auto">
     <div class="row">
-        <div class="col-11 d-block mx-auto border rounded cover-text-box">
+        <div class="col-11 d-block py-2 mx-auto border rounded cover-text-bg">
             <h1 class="text-center">{{ page_heading }}</h1>
-            <p>
-                 The Data Analytics for Canadian Climate Services (DACCS) gateway will facilitate the conversion of raw climate data
-                 from satellite observations and simulation models into relevant, credible and actionable information products.
-                 DACCS is a Science Gateway comprised of reusable modular components that will provide services to a broad scientific
-                 community through a unified user interface for: dataset search and discovery; user data management; data analytics;
-                 visualisation; workflow management; publishing; and application deployment. As part of a larger Federated Cloud
-                 Infrastructure, through this gateway, scientists can access data from different sources and process them using
-                 existing algorithms developed by the academic, government and private sectors. Typical outputs are tables, time
-                 series and graphics that support climate change impact studies, vulnerability assessments, and decision-making.
-           </p>
-         </div>
-    </div>
+            <div class="row">
+                <div class="col-5 d-block mx-auto mission-text-box">
```

Comment:
I think this looks better as two rows, not side by side columns. But that's just personal preference

Reply:
I have no preference so, changed to two rows